modules/simple_forms_api/app/controllers/simple_forms_api/v1/uploads_controller.rb

```diff
@@ -144,23 +146,12 @@ def icn
         @current_user&.icn
       end
 
-      def first_party?
-        %w[VETERAN SURVIVING_DEPENDENT].include?(params[:preparer_identification])
-      end
-
       def get_form_id
         form_number = params[:form_number]
         raise 'missing form_number in params' unless form_number
 
         FORM_NUMBER_MAP[form_number]
       end
-
-      def get_json(confirmation_number, form_id)
```

Comment:
I believe this goes along with the new code that was accidentally deleted. If I'm wrong then disregard